packages/optimizer/lib/transformers/OptimizeHeroImages.js

```diff
@@ -367,6 +370,25 @@ class OptimizeHeroImage {
     }
     appendChild(node, imgNode);
   }
+
+  isMarkedAsHeroImage(node) {
+    while (node) {
+      if (!node.tagName) {
+        node = node.parent;
```

Comment:
What if `node.parent` here is empty?

Reply:
Oh, it is missing a continue. I'll add it.